src/Helper/InvoiceHelper.php

```diff
@@ -80,4 +82,22 @@ public function isInvoiceAvailableForCurrentLoggedInCustomer(
         return null;
 
     }
+
+    /**
+     * @param Order $order
+     * @return |null
+     */
+    public function getCustomerId(Order $order)
+    {
+        $relation = $order->relations
+            ->where('referenceType', OrderRelationReference::REFERENCE_TYPE_CONTACT)
+            ->where('relation', OrderRelationReference::RELATION_TYPE_RECEIVER)
+            ->first();
```

Comment:
This should not work, as the contact id is saved in `plenty_order_main.kunden_id` and mapped when loading the order:
<img width="1008" alt="Bildschirmfoto 2020-02-12 um 09 56 58" src="https://user-images.githubusercontent.com/9987759/74318759-1c0c2f00-4d7e-11ea-833c-772d548c4eb4.png">
Please use the field `$order->contactReceiver` instead, it is mapped over an accessor method (see `Order.getContactReceiverAtribute()`.